runtime/cudaq/platform/default/rest/helpers/anyon/anyon.yml

```diff
@@ -13,6 +13,8 @@ config:
   platform-qpu: remote_rest
   # Tell NVQ++ to generate glue code to set the target backend name
   gen-target-backend: true
+  # Add preprocessor defines to compilation
+  preprocessor-defines: ["-D CUDAQ_QUANTUM_DEVICE"]
```

Comment:
Can't this preprocessor macro be added by the `nvq++` driver?

Reply:
I'm not a fan of copying the "same" configuration data/code all over the sources. It's hard to maintain and easily broken.

If you want to open an issue, that's ok.

Reply:
Done: https://github.com/NVIDIA/cuda-quantum/issues/2747